entries/normalizer.xml

```diff
@@ -13,7 +13,7 @@
 		<ul>
 			<li><p>Doesn't change the elements' value, it only changes the value used for validation.</p></li>
 			<li><p>Gets the value passed as argument, and "this" within it references the corresponding <code>DOMElement</code>.</p></li>
-			<li><p>Needs to return a String value, otherwise it will throw a <code>TypeError</code> exception.</p></li>
+			<li><p>Can return any value including <code>null</code> and <code>undefined</code>.</p></li>
```

Comment:
We should note thenversion since when this is supported

Reply:
In that case, I will wait for the next release to update it.

Reply:
Next will be a minor non-feature release

Reply:
Updated per suggestion